mariadb/README.md

```diff
@@ -4,34 +4,114 @@
 
 Based on the [Bitnami MariaDB](https://github.com/bitnami/bitnami-docker-mariadb) image for docker, this Chart bootstraps a [MariaDB](https://mariadb.com/) deployment on a [Kubernetes](http://kubernetes.io) cluster using [Helm](https://helm.sh).
 
+## TL;DR;
+
+```bash
+$ helm install mariadb
+```
+
+## Configuration
```

Comment:
We probably want to explain somewhere that you can do `helm install mariadb --set mariadbUser=blah`